app/assets/stylesheets/base/defaults.scss

```diff
@@ -383,8 +383,8 @@ p, li {
   @include paragraph;
 }
 
-h1, h2 {
-  font-family: $font-stack-heading;
+h1, h2, h3 {
+  font-family: $futura-regular;
```

Comment:
This one should be `$font-stack-heading` instead of `$futura-regular`.

Reply:
@kaylaziegler Fixed!